README.md

```diff
@@ -10,19 +10,31 @@ https://user-images.githubusercontent.com/49528805/167273992-0cbe7329-9665-4d67-
 
 1. Head up to https://dev.twitch.tv/console/extensions/create and create a new extension.
    You will need to create a new version: Select `Panel`, `Mobile` and `Video - Fullscreen` for the extension type. Leave all other settings as they are.
-2. With the version created, scroll to the bottom and click on `View on Twitch and Install`.
-   Install the extension on your channel and activate it.
-3. Enable `allow invalid certificates for resources loaded from localhost` in Chrome: [chrome://flags/#allow-insecure-localhost](chrome://flags/#allow-insecure-localhost)
-   If using Firefox, once you have started the development server, you will want to navigate to [https://localhost:8080](https://localhost:8080), click advanced and select accept the risk.
-4. Copy the `.env.sample` to `.env` (which sets `HTTPS=true`, `HOST=localhost`, and `PORT=8080`)
-5. Install dependencies for the project with `npm install`
-6. Start the development server with `npm start`
-7. To test the overlay, you will need to be live on Twitch with the extension installed.
-   If you want to use an alternate account, add the account to `Testing Account Allowlist` under the `Access` tab of the extension version and install the extension on that account.
-
-## Note
-
-CSS files are hidden through the settings.json file in .vscode
+Enable `allow invalid certificates for resources loaded from localhost` in Chrome: [`chrome://flags/#allow-insecure-localhost`](chrome://flags/#allow-insecure-localhost).
+   If using Firefox, once you have started the development server, you will want to navigate to [`https://localhost:8080`](https://localhost:8080), click advanced and select accept the risk.
+2. Copy the `.env.sample` file to `.env` (which sets `HTTPS=true`, `HOST=localhost`, and `PORT=8080`)
+3. Install dependencies for the project with `npm install`
+4. Start the development server with `npm start`
+
+## Running the extension
+
+There are two ways to run the extension. You can either add it to a channel on Twitch, or use the developer rig to test locally.
```

Comment:
recommend running directly on twitch since the dev rig is not the best experience.

Reply:
Makes senes, I had noticed that it seems to be a rather old version of Electron/Chromium